airflow/models/taskinstance.py

```diff
@@ -972,7 +1022,7 @@ def _run_raw_task(
         context = {}  # type: Dict
         actual_start_date = timezone.utcnow()
         Stats.incr('ti.start.{}.{}'.format(task.dag_id, task.task_id))
-        try:
+        try:    # pylint: disable=too-many-nested-blocks
```

Comment:
should we rather split it and extract one ore even few methodsf?  I think it's a very clear example of fairly complex code that can be simiplified by extracting methods and naming them properly

Reply:
Happy to do that if it's a problem :)

Reply:
You are right, we should :)

will do